services/localfile/localfile.proto

```diff
@@ -275,3 +281,38 @@ message SymlinkRequest {
 message MkdirRequest {
   FileAttributes dir_attrs = 1;
 }
+
+// FileFormat enum what represents file formats
+enum FileFormat {
+  UNKNOWN = 0;
```

Comment:
https://github.com/Snowflake-Labs/sansshell/pull/489#discussion_r1781431167
continuing on this.

You mentioned that the UNKNOWN file format is for the server to tell clients that it doesn't recognize the file format.
However, I don't see any server code referencing this unknown file format. How would this be used by the server?

Reply:
It for a case when we use new client version with new FileFormat, e.g. json, but old server. In this particular case it value would be casted to UNKNOWN and server reply that it doesn't support this format

Reply:
could you explain more about how the server would reply with UNKNOWN? I don't see any references to `FileFormat_UNKNOWN` in the server code.

Reply:
I was wrong. I was pretty sure that if I send from a client enum value what not exists on a server (case when client has newer version, with new enum value, then server) GRPC parser would automatically cast it to 0 and determine it as unknown. There is not the case, I'll take this in account next time we would need to add prover validation for enums to avoid such situations

But it still useful to have UNKNOWN value in a enum. e.g. for a cases when we need zero value. e.g. https://github.com/Snowflake-Labs/sansshell/blob/d15e4f94d90fa383198b76aa32252be07c527ce6/services/localfile/client/cli-controllers/file.utils.go#L35. Otherwise I would need to return some enum zero value e.g. FileFormat_YML. I could remove it, but it might be risky